src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -236,6 +274,17 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		}
 	}
 
+	getNodeIndex(n) {
+		let i = 0;
+		// walk up the node to the top and get it's index
+		n = n.previousSibling;
```

Comment:
This is fine, but FYI you can also just disable the error on a per-line basis

e.g. https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/notebook/browser/notebookExplorer/notebookSearch.ts#L535